src/ApplicationCore/Entities/FraudProtectionApiModels/SignInRequest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Dynamics.FraudProtection.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Contoso.FraudProtection.ApplicationCore.Entities.FraudProtectionApiModels
+{
+    public class SignInRequest : BaseFraudProtectionEvent
```

Comment:
Can you rename the class/file to just ```SignIn``` to be consistent with the other request models?